openshift-environment-driver/pom.xml

```diff
@@ -54,6 +54,10 @@
       <groupId>org.jboss.pnc</groupId>
       <artifactId>spi</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.pnc</groupId>
+      <artifactId>build-coordinator</artifactId>
+    </dependency>
 
```

Comment:
I don't think it is work introducing this dependency, due to configuration mess. PullingMonitor should stay in the common. I would rather made it configurable in "non-standard" way, via system property -D and env variable. Where system property overrides env if both are set.